src/luma/parser.py

```diff
@@ -195,3 +199,18 @@ def _get_signature(obj: Union[FunctionType, type], name: str) -> str:
         parameters = parameters.replace("(self, ", "(")
 
     return f"{name}{parameters}"
+
+
+def _get_param_types(obj: Union[FunctionType, type]) -> dict:
```

Comment:
I think `dict` might be an invalid type annotation for Python 3.9 (?)
```suggestion
def _get_param_types(obj: Union[FunctionType, type]) -> Dict[str, Optional[str]]:
```